### DISCUSSION TITLE
Customize demo.yaml script but not work..

### DISCUSSION THREAD

**Discussion by kronicklez** [akvorado/akvorado#1498] [Q&A]
Hi @vincentbernat ,

I'm try to adjust the current demo.yaml script by adding new destination address as per below. When i add second destination address the flow stop. May i know the script that i add below is correct or not? thanks

.demo-exporter-flows:
  - &to-v4-customers-pppoe-northern
    dst-net: 200.200.10.0/24
    dst-as: 4788
  - &to-v4-customers-pppoe-southern
    dst-net: 200.200.11.0/24
    dst-as: 4788

 routes:
   - prefixes: 200.200.10.0/24,2501:c680:cafe:1::/64
     aspath: 4788
     communities: 4788:10,4788:12
     large-communities: 4788:100:200,4788:100:201
   - prefixes: 200.200.20.0/24,2501:c680:cafe:2::/64
     aspath: 4788
     communities: 4788:10,4788:22
     large-communities: 4788:100:200,4788:100:202 

flows: &flows1
      samplingrate: 50000
      target: akvorado-inlet:2055
      flows:
        # Google
        - per-second: 1
          in-if-index: 10
          out-if-index: [20, 21]
          peak-hour: 16h
          multiplier: 3
          reverse-direction-ratio: 0.1
          <<: [*from-v4-google, *to-v4-customers-pppoe-northern, *http-src]        
        - per-second: 1
          in-if-index: 10
          out-if-index: [20, 21]
          peak-hour: 16h
          multiplier: 3
          reverse-direction-ratio: 0.1
          **<<: [*from-v4-google, *to-v4-customers-pppoe-southern, *http-src]**  ---> when i add this no flow display?

**Comment by vincentbernat** [comment]
Enclose code in backticks (```) and check the resulting YAML with the `--dump` option or with yq (see https://stackoverflow.com/questions/56588403/how-can-i-expand-yaml-anchors-and-aliases-and-get-the-resulting-yaml-in-bash)

**Reply by kronicklez** [reply]
Hi @vincentbernat ,

Is it backslash "/" or backtick "`" ? Is it below is correct? Appreciate your feedback

# Google
- per-second: 1
in-if-index: 10
out-if-index: [20, 21]
peak-hour: 16h
multiplier: 3
reverse-direction-ratio: 0.1
<<: [*from-v4-google, *to-v4-customers-pppoe-northern, *http-src]**`**----> is it like this?
- per-second: 1
in-if-index: 10
out-if-index: [20, 21]
peak-hour: 16h
multiplier: 3
reverse-direction-ratio: 0.1
**<<: [*from-v4-google, to-v4-customers-pppoe-southern, http-src]**`** --->is it like this?

**Reply by vincentbernat** [reply]
https://www.markdownguide.org/extended-syntax/#fenced-code-blocks

And see also the second part of my answer above.
